cli/repair/common.go

```diff
@@ -6,24 +6,21 @@ import (
 	"io/ioutil"
 	"os"
 	"strings"
-	"time"
 
-	goVersion "github.com/hashicorp/go-version"
+	"github.com/apex/log"
 
 	"github.com/pmezard/go-difflib/difflib"
 	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/connector"
 	"github.com/tarantool/cartridge-cli/cli/context"
 	"github.com/tarantool/cartridge-cli/cli/project"
 )
 
 var (
-	minCartridgeVersionForReload *goVersion.Version
+	minCartridgeMajorVersionForReload    = 2
```

Comment:
Reload? Is it connected with roles hot reload? (https://github.com/tarantool/cartridge/pull/1105)

Reply:
Cluster-wide config reload. This is about `repair --reload`.